livre/lib/types.h

```diff
@@ -66,7 +66,10 @@ typedef boost::scoped_ptr< TextureDataCache > TextureDataCachePtr;
 /** Map definitions */
 typedef boost::unordered_map< uint32_t, DataUploadProcessorPtr > DataUploadProcessorPtrMap;
 
-const uint32_t CONNECTION_ID( 0u );
+static const uint32_t CONNECTION_ID( 0u );
+
+typedef std::array< float, 2 > Range;
```

Comment:
Ok, same difference. Was approaching it from a float[2] angle and thought array is safer. Which one do you prefer?